ListDetail/src/main/java/com/microsoft/device/display/samples/listdetail/ui/view/ListView.kt

```diff
@@ -134,7 +127,7 @@ fun DecorativeBox(
             .clip(RoundedCornerShape(5.dp))
             .border(
                 width = 3.dp,
-                color = if (isSelected) colorResource(id = R.color.outline_blue) else Color.Transparent
+                color = if (isSelected) colorResource(id = R.color.purple_700) else Color.Transparent
```

Comment:
This is just a suggestion, up to you if you think it looks better or not! I found it easier to see the border when the width was increased to 5.dp and if `MaterialTheme.colors.primaryVariant` (or another lighter color) was used instead of purple_700